prefab/prefab-dependecy/README.md

```diff
@@ -0,0 +1,84 @@
+prefab-dependency
+==================
+
+C/C++ dependencies are C/C++ libraries packed with [Prefab] format into an AAR.
+This sample shows how to import [JsonCPP] AAR, one example of [C/C++ dependencies], from Google Maven to an Android project.
+
+[Prefab] could be enabled from Android Gradle Plugin 4.0+. In roder to use the feature, follow these 3 steps:
+
+* Enable the prefab feature in [gradle.properties]. **Note** that in 4.1+, this needs to be
+  done with `android.buildFeatures.prefab=true` in the application's build.gradle file.
```

Comment:
module's, not application's (in this case it's an application, but this advice is wrong for a user working on a library module)

Reply:
Ok, corrected.